packages/cli/src/commands/bundle/buildBundle.ts

```diff
@@ -61,7 +81,7 @@ async function buildBundle(
     sourceMapUrl,
     dev: args.dev,
     minify: args.minify !== undefined ? args.minify : !args.dev,
-    platform: args.platform,
```

Comment:
Let's revert this change, since we use `arg.platform` everywhere in this function

Reply:
Its has been assigned on line 48  
```
const platform: string = args.platform || '';
```

Reply:
That's what I refer to. It's only assigned there and used here, but other parts of this function rely on `args.platform` directly. No point in changing that for now :)